tools/clang/test/SemaHLSL/hlsl/objects/HitObject/hitobject-unavailable-pre-sm69.hlsl

```diff
@@ -1,20 +1,9 @@
 // RUN: %dxc -T lib_6_8 %s -verify
 
 // Check that the HitObject type name of Shader Execution Reordering is unclaimed pre SM 6.9.
-// expected-no-diagnostics
-
-namespace dx {
-struct HitObject {
-  int notTheSM69HitObject;
-  static HitObject MakeNop() {
-    HitObject hit;
-    hit.notTheSM69HitObject = 1;
-    return hit;
-  }
-};
-}
 
 [shader("raygeneration")]
 void main() {
+  // expected-warning@+1{{potential misuse of built-in function 'dx::HitObject::MakeNop' in shader model lib_6_8; introduced in shader model 6.9}}
```

Comment:
Not sure if HitObject::Miss is in yet or not, but I imagine a similar test would need to be added in that case. Just a comment FYI, no resolution needed.